_pages/clients/curl.md

```diff
@@ -8,6 +8,7 @@ show_related: true
 image:
     teaser: /language_logos/curl.svg
 redirect_from:
+  - /clients/cURL/
```

Comment:
The redirect from value is correct, but this is in the wrong file. This should be added to `_pages/clients.md`.

Reply:
> @bnlb All trailing slashes have been added. Mel* did go ahead and approve the ink for /nltk.md - her response can be found in the google sheet.

Didn't realize you updated `/clients/cURL/` to be on the clients page. Looks good!